doc/source/tutorial/stats/qmc_latin_hypercube.md

```diff
@@ -0,0 +1,65 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
```

Comment:
This file is now an orphan, and I added it to a See Also section under QMC. Do you have better ideas for where to put it?

Reply:
Maybe I don't understand - how is it an orphan if it's referenced from somewhere?

Reply:
It is referenced, but not in a table of contents. The major visual difference is that the page won't have a left sidebar, and the breadcrumbs at the top will be empty:

![Captura de imagem_20240628_181235](https://github.com/scipy/scipy/assets/3949932/98443ef7-e44b-46fa-ac2e-ad855add9ffa)

Reply:
> Let me know if you want me to look at something specific.

@tupui can you suggest how/whether this should be worked into the QMC tutorial so it's not an orphan? (Or it can be moved back to the API docstring, if you prefer.)

Reply:
There could be a section at the end of the QMC tutorial showing a real life example, but that might seem odd and I would not want people to default to LHS actually. I think I would prefer to get this back in the API docstring as it's a very short example.